packages/core/contracts/cross-chain-oracle/README.md

```diff
@@ -0,0 +1,68 @@
+# Deployments
+
+## Setup
+
+Before running anything below, make sure you have run `yarn build` from the root of the repo.
+
+Note that the fork network instructions can be used as a test run prior to the public network
+deployments.
+
+## Starting a fork
+
+To run a forked network deployment, run the following command:
+
+```sh
+HARDHAT_CHAIN_ID=<FORKED_CHAIN_ID> yarn hardhat node --fork <URL> --no-deploy --port <PORT>
+```
+
+You'll need to run two forks in separate terminals on different ports to do the deployments below.
+
+Note: in the commands below, you'll need to set the relevant `NODE_URL_X` environment variable to the url of the locally forked network `http://localhost:<PORT>`.
+
+If you're having trouble redeploying contracts because `hardhat` wants to "reuse" contracts, then run `yarn` in the `core` package to reset `deployments`.
+
+## Arbitrum
+
+1. Start by exporting some environment variables (or storing them in a .env):
+
+```sh
+export NODE_URL_1=<MAINNET_URL>
+export NODE_URL_42161=<ARBITRUM_URL>
+# When running against a forked network, set the URL to http://localhost:<PORT>
```

Comment:
nit: should this comment go above the NODE_URL lines?

Reply:
Yeah done